### ISSUE TITLE
Error at Sphinx initialization (failed to rename)

### ISSUE THREAD

**Issue by donatelloOo** [trustin/sphinx-maven-plugin#4]
We are facing this issue (a bit randomly) since upgrade to v1.7.0.
Note that it happens when running this on Windows (not sure about Linux).
My 2 cents bet is it could be linked to the Windows file handler, which often leads to this kind of issue.

```console
$ mvn clean install

[INFO] --- sphinx-maven-plugin:1.7.0:generate (rtd) @ documentation ---
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time: 31.959 s
[INFO] Finished at: 2018-02-23T15:21:49+01:00
[INFO] Final Memory: 66M/1667M
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal kr.motd.maven:sphinx-maven-plugin:1.7.0:generate (rtd) on project documentation: Failed to run the report: Failed to rename: C:\DEV\workspace-trunk\myproject\documentation\target\sphinx\1.7.0.tmp.1878009430852915223 -> 1.7.0 -> [Help 1]
```

We are using following configuration, but I doubt there is something useful here:
```xml
            <plugin>
                <!-- https://trustin.github.io/sphinx-maven-plugin/ -->
                <!-- https://github.com/trustin/sphinx-maven-plugin -->
                <groupId>kr.motd.maven</groupId>
                <artifactId>sphinx-maven-plugin</artifactId>
                <version>${sphinx-maven-plugin-version}</version>
                <executions>
                    <execution>
                        <id>rtd</id>
                        <phase>package</phase>
                        <goals>
                            <goal>generate</goal>
                        </goals>
                        <configuration>
                            <sourceDirectory>${project.build.directory}/classes</sourceDirectory>
                            <configDirectory>${project.build.directory}/classes</configDirectory>
                            <outputDirectory>${project.build.directory}/rtd</outputDirectory>
                            <warningsAsErrors>${failOnWarnings}</warningsAsErrors>
                        </configuration>
                    </execution>
                </executions>
            </plugin>
```

**Comment by trustin** [comment]
Sorry for a late response. I've just release `sphinx-maven-plugin-2.0.0` which uses completely different mechanism. Could you try it and let me know if it fixes your problem?

**Comment by trustin** [comment]
Just released 2.0.2 which fixes some issues on Windows. Now I'm sure it works fine on Windows. Please open a new issue if you still have a problem with 2.0.2.
